exercises/luhn/canonical-data.json

```diff
@@ -1,83 +1,109 @@
 {
   "exercise": "luhn",
-  "version": "1.0.1",
+  "version": "1.1.0",
   "cases": [
     {
       "description": "single digit strings can not be valid",
       "property": "valid",
-      "input": "1",
+      "input": {
+        "ccNumber": "1"
```

Comment:
I assume `ccNumber` stands for `creditCardNumber`. If so, why not write it out and use `creditCardNumber` instead?

Reply:
No reason not to other than the description.md also says it could be used for Canadian Social Insurance Numbers.  I didn't want the field to be overly long and I couldn't come up with a good, meaningful, name that was generic enough for this first commit. 

Yes, `ccNumber` stands for `creditCardNumber`.

Reply:
Aha, I see. Maybe we should then revert to something plain like "input"?

Reply:
"input" inside of "input"? 👎 Technically, I guess, still satisfies #996.  How about `"aValue"`?

Reply:
I don't really like the "a" prefix. How about just "value"?

Reply:
Sold!  Thanks @ErikSchierboom.